openassessment/templates/openassessmentblock/staff_area/oa_staff_override_assessment.html

```diff
@@ -3,25 +3,36 @@
 
     <div class="wrapper--staff-assessment">
         <div class="step__instruction">
-            <p>{% trans "Override this learner's current grade using the problem's rubric." %}</p>
+            {% if not is_team_assignment %}
+                <p>{% trans "Override this learner's current grade using the problem's rubric." %}</p>
+            {% else %}
+                <p>{% trans "Override this team's current grade using the problem's rubric." %}</p>
+            {% endif %}
         </div>
 
         <div class="step__content">
             <article class="staff-assessment">
                 <div class="staff-assessment__display">
                     <header class="staff-assessment__display__header">
                         <h3 class="staff-assessment__display__title">
-                            {% if student_username %}
-                                {% blocktrans %}
-                                    Response for: {{ student_username }}
-                                {% endblocktrans %}
+                            {% if not is_team_assignment %}
+                                {% if student_username %}
+                                    {% blocktrans %}
+                                        Response for: {{ student_username }}
+                                    {% endblocktrans %}
+                                {% else %}
+                                    {% trans "Learner Response" %}
+                                {% endif %}
                             {% else %}
-                                {% trans "Learner Response" %}
```

Comment:
is there an easy way to potentially show the team name here?

Reply:
^ Same as [thread](https://github.com/edx/edx-ora2/pull/1439#discussion_r441633073). 

Did this recently in the "Manage individual learners" pane